### PR TITLE
Fix details modal

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

- There were several fields missing from our study details modal. This PR re-adds them.
- Removed unused test data
- Refactored http client 

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).

image

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Local smoke testing on the details modal on the data table and general regression testing

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.

https://github.com/serotracker/iit-backend/pull/178

### CODE REVIEW

File: src/httpClient.ts [hunk 1]
```diff
@@ -129,27 +128,24 @@ export default class httpClient {
...
             sampling_start_date: startDate,
             sampling_end_date: endDate,
             sorting_key: sorting_key,
             reverse: reverse,
             per_page: null,
             page_index: null,
-            columns: only_explore_columns ? explore_columns : all_columns,
         }
+
+        if( only_explore_columns ){
```
Comment (annoyingpr) [inline src/httpClient.ts:145]: Why this instead of `columns: only_explore_columns ? explore_columns : all_columns,`?

File: src/httpClient.ts [hunk 2]
```diff
@@ -174,14 +169,18 @@ export default class httpClient {
...
                 source_type: item.source_type,
                 specimen_type: Array.isArray(item.specimen_type) ? item.specimen_type : [item.specimen_type],
                 state: item.state,
                 study_type: item.study_type,
-                test_type: item.study_type,
+                summary: item.summary,
+                test_type: item.test_type,
+                test_manufacturer: item.test_manufacturer,
                 url: item.url
             };
```
Comment (annoyingpr) [inline src/httpClient.ts:185]: Is it possible to use object destructuring for this? Check out these [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).

**Review by annoyingpr — APPROVED** [review]
🚀  - great work, just a couple of comments!

File: src/httpClient.ts (hunk 1 above)
Comment (atatmaja) [reply src/httpClient.ts:145]: If you don't supply the "columns" parameter the backend will supply all columns by default.

It's better if we don't have to update the supplied columns in both the front end and the backend, I think that was the reason why the study details modal got out of sync in the first place.

File: src/httpClient.ts (hunk 2 above)
Comment (atatmaja) [reply src/httpClient.ts]: Typescript will make this a bit tricky I feel. I think a better long term solution is to find a proper way to map an http response to a typescript object without being so verbose. I will note this in a TODO comment.

**Pull request merged**